build_defs/python.build_defs

```diff
@@ -513,9 +513,9 @@ def python_wheel(name:str, version:str, labels:list=[], hashes:list=None, packag
                  outs:list=None, post_install_commands:list=None, patch:str|list=None, licences:list=None,
                  test_only:bool&testonly=False, repo:str=None, zip_safe:bool=True, visibility:list=None,
                  deps:list=[], name_scheme:str|list=CONFIG.PYTHON.WHEEL_NAME_SCHEME,
-                 strip:list=['*.pyc', 'tests'], binary = False, entry_points={},
+                 strip:list=['*.pyc', 'tests'], binary = False, entry_points:dict={},
                  tool:str=CONFIG.PYTHON.WHEEL_TOOL, prereleases:bool=CONFIG.PYTHON.PRERELEASES,
-                 tool_verbosity:str=CONFIG.PYTHON.VERBOSITY):
+                 tool_verbosity:str=CONFIG.PYTHON.VERBOSITY, interpreter:str=CONFIG.PYTHON_DEFAULT_INTERPRETER):
```

Comment:
```suggestion
                 tool_verbosity:str=CONFIG.PYTHON.VERBOSITY, interpreter:str=CONFIG.PYTHON.DEFAULT_INTERPRETER):
```

Reply:
yup thanks, every single test failed which also pointed to this :)